javascript/operators/nullish_coalescing.json

```diff
@@ -44,11 +44,11 @@
               "version_added": false
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "80"
```

Comment:
Hmm, looks like ChromeStatus doesn’t mention WebView support?

Reply:
Given they all use the v8 engine, I would be very surprised if this would be different in Webview.

Reply:
I've noticed that ChromeStatus has a separate part for WebView features, though if it's a JavaScript feature then it's probably a data issue in ChromeStatus.  Sorry, didn't realize this was JavaScript data!